.github/workflows/cypress.yml

```
@@ -0,0 +1,108 @@
+name: Running Cypress Tests
+
+on:
+ pull_request:
+   branches: [main, develop]
+
+
+jobs:
+
+ Setup-Run-Cypress-Tests:
+   timeout-minutes: 25
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 3

(indentation)